v2/cli/term/event_page.go

```diff
@@ -35,14 +37,28 @@ func newEventPage(
 		eventInfo:  tview.NewTextView().SetDynamicColors(true),
 		workerInfo: tview.NewTextView().SetDynamicColors(true),
 		jobsTable:  tview.NewTable().SetSelectable(true, false),
+		eventLogs:  tview.NewTextView().SetDynamicColors(true),
 		usage: tview.NewTextView().SetDynamicColors(true).SetText(
-			"[yellow](F5) [white]Reload    [yellow](<-/Del) [white]Back    [yellow](ESC) [white]Home    [yellow](Q) [white]Quit", // nolint: lll
+			"[yellow](F5) [white]Reload    [yellow](<-/Del) [white]Back    [yellow](L) [white]Logs    [yellow](ESC) [white]Home    [yellow](Q) [white]Quit", // nolint: lll
 		),
 	}
 	e.eventInfo.SetBorder(true).SetBorderColor(tcell.ColorYellow)
 	e.workerInfo.SetBorder(true).SetBorderColor(tcell.ColorYellow).
 		SetTitle("Worker")
 	e.jobsTable.SetBorder(true).SetTitle("Jobs")
+	e.eventLogs.SetBorder(true).SetTitle("Logs (<-/Del) Quit")
+
+	// Returns a new primitive which puts the provided primitive in the center and
+	// sets its size to the given width and height.
+	e.logModal = tview.NewFlex().
+		AddItem(nil, 0, 1, false).
+		AddItem(tview.NewFlex().
+			SetDirection(tview.FlexRow).
+			AddItem(nil, 0, 1, false).
+			AddItem(e.eventLogs, 25, 1, false).
+			AddItem(nil, 0, 1, false), 85, 1, false).
+		AddItem(nil, 0, 1, false)
+
```

Comment:
A lot of this smells reusable to me.

I think we should extract this stuff into its own component that can be used across the event page (for displaying worker logs) and the job page (for displaying job logs).

I know that as things stand right now, the job page shows logs in the "table area" below the job details. This _probably_ isn't the right way to go.

Jobs can be composed of multiple containers. We should probably use that table area to list the job's containers. Selecting a container can pop up the logs for that container-- which leads me back to the idea that what you've done here on the event page is something we should consider making reusable for both the event and job page.

In fact...

Let's put a pin in this PR for now and have you focus on a new PR that lists a job's containers in the table area of the job page (selecting one of them doesn't need to do anything for now). Once we get that in, then we can revistiv this PR and figure out how best to make this little log pop-up reusable.